neural_lam/models/ar_model.py

```diff
@@ -262,7 +262,7 @@ def unroll_prediction(self, init_states, forcing_features, true_states):
                 pred_std_list, dim=1
             )  # (B, pred_steps, num_grid_nodes, d_f)
         else:
-            pred_std = self.per_var_std  # (d_f,)
+            pred_std = self.diff_std  # (d_f,)
```

Comment:
This seems unrelated and just an artifact from merging/rebasing.

Reply:
Fixed with https://github.com/mllam/neural-lam/pull/77/commits/ae69f3f32d0add5a47db1edefaf8de36129ff58d